pkg/endpoints/endpoints.go

```diff
@@ -0,0 +1,111 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package endpoints
+
+import (
+	"errors"
+	"net/url"
+	"strings"
+)
+
+// ServiceEndpointFormat is used to identify the custom endpoint of IBM Cloud service.
+var ServiceEndpointFormat string
+
+const (
+	// VPCServiceID used to identify VPC service.
+	VPCServiceID = "vpc"
+	// PVSServiceID used to identify PowerVS service.
+	PVSServiceID = "powervs"
+	// RCServiceID used to identify Resource-Controller service.
+	RCServiceID = "rc"
```

Comment:
you can create a `ServiceID` type for these entries

Reply:
Noted, will do the required changes.